tensorboard/webapp/plugins/npmi/npmi_plugin.py

```diff
@@ -126,28 +129,28 @@ def values_impl(self):
         for (run, _) in six.iteritems(mapping):
             event = self._multiplexer.Tensors(run, "metric_results")[0]
             event_data = tensor_util.make_ndarray(event.tensor_proto).tolist()
-            result[run] = {"values": event_data}
-        contents = json.dumps(result)
-        # This is ugly, but there should never be a NaN string to break this
-        contents = contents.replace("NaN", "null")
+            result[run] = event_data
+        contents = json.dumps(result, cls=safe_encoder.SafeEncoder)
         return contents
 
-    @wrappers.Request.application
+    @ wrappers.Request.application
```

Comment:
Looks like an extra space got added between `@` and `wrappers` here?

Reply:
Yes, I have problems with getting the linter+autoformat to work correctly somehow... Fixed in [74b1fea](https://github.com/tensorflow/tensorboard/pull/3799/commits/74b1feab18b3dfa001a12a74141c643409a7c6b1)